plugins/parsers/csv/parser.go

```diff
@@ -148,6 +149,35 @@ outer:
 				}
 			}
 
+			// In case if defined column names & types count don't match.
+			if i < len(p.ColumnTypes) {
```

Comment:
Seems like throwing an error if they don't match before here would be a good idea.

Reply:
i've added the throw error statement on `newCSVParser`, seems like it fits there better ?